src/pyuvsim/simsetup.py

```diff
@@ -2019,6 +2027,9 @@ def initialize_uvdata_from_params(
             *tele_params["telescope_location"], unit="m"
         )
     elif tele_params["telescope_frame"] == "mcmf":
+        # to get here, lunarsky has to be installed, so don't need to test for it
+        from lunarsky import MoonLocation
```

Comment:
Not that I don't believe you in this comment, But i cannot actually find _where_ we have already checked for Lunarsky here. If we're making a mock catalog it gets called there but what about the case where we don't?